common/djangoapps/student/views/management.py

```diff
@@ -741,6 +742,13 @@ def create_account_with_params(request, params):
     else:
         compose_and_send_activation_email(user, profile, registration)
 
+    if settings.FEATURES.get('AUTO_SOFTWARE_SECURE_PHOTO_VERIFICATION'):
+        SoftwareSecurePhotoVerification.objects.create(
+            status="approved",
+            user_id=user.id,
+            submitted_at=datetime.datetime.now(UTC),
+            photo_id_key=uuid.uuid4())
```

Comment:
It is ok to fill in `uuid.uuid4()`? Can this affect another modules or something else?

Reply:
You're right, I will change it to the format that is used on photo verification